pgctl/cli.py

```diff
@@ -410,7 +410,10 @@ def debug(self):
                 'Must debug exactly one service, not: ' + commafy(self.service_names),
             )
 
-        self.stop()
+        if service.state['state'] != 'down':
+            self.stop()
```

Comment:
Are the "Stopping ..." "Stopped ..." messages not equally confusing and useless when debugging?